odoo/addons/base/views/res_partner_views.xml

```diff
@@ -122,53 +122,41 @@
                 <sheet>
                     <div class="oe_button_box" name="button_box"/>
                     <widget name="web_ribbon" title="Archived" bg_color="text-bg-danger" invisible="active"/>
-                    <field name="image_1920" widget='image' class="oe_avatar" options='{"preview_image": "avatar_128"}'/>
+                    <field name="image_1920" widget="partner_image" class="oe_avatar position-absolute top-0 end-0" options='{"preview_image": "avatar_128"}'/>
                     <div class="oe_title mb24">
                         <field name="company_type" widget="radio" options="{'horizontal': true}"/>
                         <h1>
                             <field id="company" options="{'line_breaks': False}" widget="text" class="text-break" name="name" default_focus="1" placeholder="e.g. Lumber Inc" invisible="not is_company" required="type == 'contact'"/>
                             <field id="individual" options="{'line_breaks': False}" widget="text" class="text-break" name="name" default_focus="1" placeholder="e.g. Brandon Freeman" invisible="is_company" required="type == 'contact'"/>
                         </h1>
-                        <div class="o_row">
-                            <field name="parent_id"
-                                widget="res_partner_many2one"
-                                placeholder="Company Name..."
-                                domain="[('is_company', '=', True)]" context="{'default_is_company': True, 'show_vat': True, 'default_user_id': user_id}"
-                                invisible="((is_company and not parent_id) or company_name) and company_name != ''"/>
-                                <field name="company_name" invisible="not company_name or company_name == '' or is_company"/>
-                                <button name="create_company" icon="fa-plus-square" string="Create company"
-                                    type="object" class="oe_edit_only btn-link"
-                                    invisible="is_company or company_name == '' or not company_name"/>
-                        </div>
+                        <field name="function" placeholder="e.g. Sales Director" invisible="is_company"/>
                     </div>
-
                     <group>
                         <group>
+                            <field name="parent_id" string="Company"
+                                widget="res_partner_many2one"
+                                placeholder="Company Name..."
+                                domain="[('is_company', '=', True)]" context="{'default_is_company': True, 'default_user_id': user_id}"
+                                invisible="((is_company and not parent_id) or company_name) and company_name != ''"/>
+                            <field name="company_name" invisible="not company_name or company_name == '' or is_company"/>
                             <span class="o_form_label o_td_label o_address_type" name="address_name">
```

Comment:
Would also keep the button if you can manage to fit it in somehow (or maybe below?), otherwise it'd be difficult to convert the "company name" into an actual company.